src/VisualStudio/TestUtilities2/CallHierarchy/CallHierarchyTestState.vb

```diff
@@ -172,19 +172,30 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.CallHierarchy
         End Sub
 
         Friend Sub VerifyResultName(root As CallHierarchyItem, searchCategory As String, expectedCallers As String(), Optional scope As CallHierarchySearchScope = CallHierarchySearchScope.EntireSolution, Optional documents As IImmutableSet(Of Document) = Nothing)
+            Dim callers = New List(Of String)
             SearchRoot(root, searchCategory, Sub(c As ICallHierarchyNameItem)
-                                                 Assert.Contains(ConvertToName(c), expectedCallers)
+                                                 callers.Add(ConvertToName(c))
                                              End Sub,
                 scope,
                 documents)
+
+            For Each expected In expectedCallers
```

Comment:
worth adding a check that the size of the two collections is the same as well to make sure nothing is added that wasn't expected